### PR TITLE
expose heating status/current implements #420 fixes #433

### LINKED ISSUE #433: Heating switch not showing correct status
I gave your latest update a try.
I enabled heating via the app and then started the esp.
The heating switch is turn off but it is activated in the app.
jk-heating

Hardware version: V11.XW
Software version: V11.287H
Model: B1A8S10P

### PR DESCRIPTION
**Pull request by RFDarter**
implements #420
fixes #433

### CODE REVIEW

File: components/jk_bms_ble/jk_bms_ble.h [hunk 1]
```diff
@@ -152,6 +156,9 @@ class JkBmsBle : public esphome::ble_client::BLEClientNode, public PollingCompon
   void set_emergency_time_countdown_sensor(sensor::Sensor *emergency_time_countdown_sensor) {
     emergency_time_countdown_sensor_ = emergency_time_countdown_sensor;
   }
+  void set_heating_current_sensor(sensor::Sensor *heating_current_sensor) {
+    heating_current_sensor_ = heating_current_sensor;
```
Comment (RFDarter) [inline components/jk_bms_ble/jk_bms_ble.h:160]: Not sure what is worng here on the format

File: components/jk_bms_ble/jk_bms_ble.cpp [hunk 2]
```diff
@@ -556,6 +558,13 @@ void JkBmsBle::decode_jk02_cell_info_(const std::vector<uint8_t> &data) {
   //           0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00
   //           0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00
   //           0x00
+
+  // 224   1   0x01                   Heating status          0x00: off, 0x01: on
+  this->publish_state_(this->heating_binary_sensor_, (bool) data[224 + offset]);
```
Comment (syssi) [inline components/jk_bms_ble/jk_bms_ble.cpp:563]: At the moment we cannot be sure this byte position applies to `JK02_32S` frames too. All your devices are so called old style 24S devices?

Comment (RFDarter) [reply components/jk_bms_ble/jk_bms_ble.cpp:563]: Yep, unfortunately i dont have a new style here to test.

Comment (syssi) [reply components/jk_bms_ble/jk_bms_ble.cpp:563]: I've a BMS here using the new protocol version. It looks like old version. IMO just the firmware (+ protocol version) has changed. My model doesn't have an heater element but I will check the `heating current` and `status`. In best case it's `0`.

**Pull request merged**